src/app/home-page/new-dashboard/new-dashboard.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { Base } from 'app/shared/base';
+import { Observable } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { User } from '../../shared/swagger/model/user';
+import { UserQuery } from '../../shared/user/user.query';
+import { RegisterToolService } from 'app/container/register-tool/register-tool.service';
+import { MatDialog } from '@angular/material/dialog';
+import { RegisterToolComponent } from 'app/container/register-tool/register-tool.component';
+import { AlertService } from 'app/shared/alert/state/alert.service';
+
+@Component({
+  selector: 'app-new-dashboard',
+  templateUrl: './new-dashboard.component.html',
+  styleUrls: ['./new-dashboard.component.scss'],
+})
+export class NewDashboardComponent extends Base implements OnInit {
+  public user$: Observable<User>;
+  user: User;
```

Comment:
It doesn't look like you're using `user` nor `$user`. And if you don't need them, you probably don't need `userQuery` neither, not the block on lines 31-33.

Reply:
Oh yes